pharmacy-ui/src/views/Drug/Drug.scss

```diff
@@ -0,0 +1,22 @@
+.wrapper {
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    grid-template-areas: 'sales drug'; 
+  }
```

Comment:
you're defining only one column in the template-columns, and 2 in the template-areas

Reply:
I thought of adding another component for second column. But, then dropped the idea....for future. I will remove the second column. Thanks for catching it.